pkg/config/leaderelection/leaderelection.go

```diff
@@ -85,13 +85,21 @@ func LeaderElectionDefaulting(config configv1.LeaderElection, defaultNamespace,
 	ret := *(&config).DeepCopy()
 
 	if ret.LeaseDuration.Duration == 0 {
-		ret.LeaseDuration.Duration = 60 * time.Second
+		// account for 20% jitter of the RetryPeriod
+		ret.LeaseDuration.Duration = 99 * time.Second
 	}
+	// these total 30s and then 20% jitter on RetryPeriod.  This gives us two properties
+	//  1. We can span a 60s kube-apiserver communication interruption (30+jitter1 + 30+jitter2) is always at least 60s.
+	//  2. We can always retry at least three times inside of our leaseDuration even in cases of long jitter and long
+	//     waits (black hole) that causes us to bump the RenewDeadline.  In cases of a short renew failure, we may retry
+	//     more often.
 	if ret.RenewDeadline.Duration == 0 {
-		ret.RenewDeadline.Duration = 35 * time.Second
+		// we choose a very long RenewDeadline so that case of very slow etcd are exacerbated by lost leases and
+		// cascading retries.
+		ret.RenewDeadline.Duration = 17 * time.Second
```

Comment:
Note if you do this you only get once chance to hold the lease, so I would expect this to make mots leases fail.  Having LeaseDeadline >>> LeaseDuration makes no sense.  You want to have about 10-20s of slack between RenewDeadline and LeaseDuration.  I.e.:

99 lease, 70 renew, retry period 34s makes some sense (You get two tries)
99 lease, 70 renew, retry period 60s makes sense (we try one time)